src/collective/glossary/profiles/default/types/Term.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Basic metadata -->
   <property name="title" i18n:translate="">Term</property>
   <property name="description" i18n:translate="">A Term</property>
-  <property name="content_icon">++resource++collective.glossary/term-icon.png</property>
+  <property name="content_icon">++resource++collective.glossary/img/term-icon.png</property>
```

Comment:
for some reason the portlet navigation don't find the icon in this new url (after run upgrade step).

need help.

Reply:
are you sure icons are generated inside the `img` directory?

Reply:
yes:
https://github.com/collective/collective.glossary/pull/45/files#diff-28b51d16260380d9fae7857a4ba6dbeaR21
![image](https://user-images.githubusercontent.com/131209/58288388-d2139100-7d89-11e9-9a3f-ba5097db0862.png)